pkg/replica/server.go

```diff
@@ -19,15 +19,19 @@ type Server struct {
 	backing                   *backingfile.BackingFile
 	revisionCounterDisabled   bool
 	unmapMarkDiskChainRemoved bool
+	snapshotMaxCount          int
+	SnapshotMaxSize           int64
```

Comment:
Why exported?

Reply:
I updated naming and didn't notice that. Thank you.